### ISSUE TITLE
Zero Transactions To Import

### ISSUE THREAD

**Issue by dfredm** [aniav/ynab-csv#23]
We could not find any valid transactions to import from the files provided.

**Comment by aniav** [comment]
Could you provide an example file you try to upload (sensitive data should be cleaned)?  Without it I can't even try fixing the issue. Thanks!

**Comment by jurecuhalev** [comment]
I have the same problem, input CSV:

```
"Date","Payee","Account number","Transaction type","Payment reference","Category","Amount (EUR)","Amount (Foreign Currency)","Type Foreign Currency","Exchange Rate"
"2019-10-15","ZITO DUNAJSKA","","MasterCard Payment","","Food & Groceries","-2.2","-2.2","EUR","1.0"
"2019-10-15","CIN CIN FOOD","","MasterCard Payment","","Bars & Restaurants","-4.6","-4.6","EUR","1.0"
```

Output CSV:

```
"Date","Payee","Memo","Outflow","Inflow"
"2019-10-15","ZITO DUNAJSKA","","",""
"2019-10-15","CIN CIN FOOD","","",""
```

It seems it's missing the actual amounts information.

**Comment by MaximBazarov** [comment]
I have the same issue

**Comment by MaximBazarov** [comment]
The solution basically is to select appropriate columns in the interface

**Comment by aniav** [comment]
So I just tested the provided CSV file and it came out properly after I selected these columns. Note I selected the same column for both Outflow and Inflow.

Output CSV content:
```
"Date","Payee","Memo","Outflow","Inflow"
"2019-10-15","ZITO DUNAJSKA","","2.2",""
"2019-10-15","CIN CIN FOOD","","4.6",""
```

Is this fine?

<img width="1158" alt="Screenshot 2019-11-11 at 18 26 07" src="https://user-images.githubusercontent.com/182546/68607364-bf645400-04b0-11ea-97ff-66766563d9a7.png">
